Directory.Build.targets

```diff
@@ -12,28 +12,33 @@
   <ItemGroup>
     <!--3rd party dependencies-->
     <PackageReference Update="Microsoft.NET.Test.Sdk" Version="16.11.0" />
```

Comment:
`ctrl + k + d` to make the formatting pretty?

Reply:
It si correctly formated on my machine 😲

Reply:
Tabs vs spaces

Reply:
Mhh, I thought we had a file somewhere that alignes these settings for all of us 🤔

Reply:
But this either is not part of the repo or GitHub isn't aware. The root cause here is, that some lines are indented by spaces, some by tabs and some by a mix of both.

Reply:
unified that ✅